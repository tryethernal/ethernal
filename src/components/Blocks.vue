<template>
    <v-container fluid>
        <Block-List />
    </v-container>
</template>

<script>
import BlockList from './BlockList';

export default {
    name: 'Blocks',
    components: {
        BlockList
    }
}
</script>
