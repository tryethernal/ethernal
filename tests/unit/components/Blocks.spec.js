import MockHelper from '../MockHelper';

import Blocks from '@/components/Blocks.vue';

describe('Blocks.vue', () => {
    let helper;

    beforeEach(() => {
        helper = new MockHelper();
    });

    it('Should show the blocks list', async () => {
        const wrapper = helper.mountFn(Blocks, {
            stubs: ['Block-List']
        });
        await new Promise(process.nextTick);

        expect(wrapper.html()).toMatchSnapshot();
    });
});
