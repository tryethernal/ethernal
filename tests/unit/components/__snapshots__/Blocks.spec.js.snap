// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blocks.vue Should show the blocks list 1`] = `
<div class="container container--fluid">
  <block-list-stub></block-list-stub>
</div>
`;
